hoot-core-test/src/test/cpp/hoot/core/io/ServiceOsmApiDbBulkWriterTest.cpp

```diff
@@ -345,7 +345,8 @@ class ServiceOsmApiDbBulkWriterTest : public CppUnit::TestFixture
 
     //init db
     ServicesDbTestUtils::deleteDataFromOsmApiTestDatabase();
-    const QString scriptDir = "${HOOT_HOME}/test-files/servicesdb";
+//    const QString scriptDir = "${HOOT_HOME}/test-files/servicesdb";
+    const QString scriptDir = QString(getenv("HOOT_HOME")) + "/test-files/servicesdb";
```

Comment:
Did we determine this method of getting HOOT_HOME was fixing the core errors on a CentOS install from RPMs?  If so and we want to make this change, recommend removing old test line.  Same goes for other instances below.